pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/ResourceQuotasBase.java

```diff
@@ -87,13 +84,9 @@ protected void internalSetNamespaceBundleResourceQuota(String bundleRange, Resou
         NamespaceBundle nsBundle = validateNamespaceBundleRange(namespaceName, policies.bundles, bundleRange);
 
         try {
-            pulsar().getLocalZkCacheService().getResourceQuotaCache().setQuota(nsBundle, quota);
+            pulsar().getBrokerService().getBundlesQuotas().setResourceQuota(nsBundle, quota).join();
             log.info("[{}] Successfully set resource quota for namespace bundle {}", clientAppId(),
                     nsBundle.toString());
-        } catch (KeeperException.NoNodeException e) {
-            log.warn("[{}] Failed to set resource quota for namespace bundle {}: concurrent modification",
-                    clientAppId(), nsBundle.toString());
-            throw new RestException(Status.CONFLICT, "Concurrent modification on namespace bundle quota");
```

Comment:
are we losing this CONFLICT case ?
I believe it is not a big deal, so this change is good.
But I would like to see your opinion @merlimat

Reply:
The message and return code are actually wrong, and we cannot have a `NoNodeException` in this case because the write will use set or create depending on the case.